fastlane/Fastfile

```diff
@@ -835,12 +789,7 @@ platform :ios do
   #
   #####################################################################################
   lane :trigger_release_build do |options|
-    buildkite_trigger_build(
-      buildkite_organization: 'automattic',
-      buildkite_pipeline: 'wordpress-ios',
-      branch: options[:branch_to_build],
-      pipeline_file: 'release-builds.yml'
-    )
+    trigger_buildkite_release_build(branch: options[:branch_to_build], beta: false)
```

Comment:
I also considered ditching the two lanes for a single one with a `beta` parameter. I decided to keep them because it can happen to have to start one such build from the terminal and in that case it's handy to have dedicated lanes and tab complete between the two.